src/test/regress/citus_tests/config.py

```diff
@@ -184,6 +175,7 @@ def __init__(self, arguments):
         self.user = SUPER_USER_NAME
         self.mixed_mode = arguments["--mixed"]
         self.fixed_port = 57635
+        self.is_mx = False
```

Comment:
can you open an issue saying that we should evaluate enabling MX on upgrade tests?